contracts/treasury/LiquidationStrategyBase.sol

```diff
@@ -245,9 +248,10 @@ abstract contract LiquidationStrategy is ILiquidationStrategy, PermissionAdmin,
   function isLiquidationEnabledAt(uint256 timestamp) public override view returns (bool) {
     if (timestamp < block.timestamp) return false;
     LiquidationSchedule memory schedule = _liquidationSchedule;
+    if (schedule.repeatedPeriod == 0) return false;
     if (timestamp < uint256(schedule.startTime)) return false;
     uint256 timeInPeriod = (timestamp - uint256(schedule.startTime)) % uint256(schedule.repeatedPeriod);
-    return timeInPeriod < schedule.duration;
+    return timeInPeriod <= schedule.duration;
```

Comment:
Another way to pause this contract is setting schedule.duration = 0
So we don't need another SLOAD when liquidating
Moreover, only admin can unpause this contract

Reply:
So `timeInPeriod < schedule.duration` makes sense if we do this way.

Reply:
yeah we can move to check duration == 0, I added `pause` which allows operator to make it more flexible to react, since it takes 2 weeks to vote + execute, which is too slow

Reply:
setting liquidation schedule should be done via a vote, but pause/unpause may not

Reply:
Pausing is rarely happened. When it happens, it takes time to make sure there is no bug in the contract.
And it makes more sense that msig is admin of this contract

Reply:
By design, the admin of this contract is (most likely) the executor, while the daoOperator can be the operator